samples/DataSourceWithProcessingAndExtensions/SampleDriver/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.Toolkit.Engine;
+using SampleExtensions.CompositeCookers;
+using SampleExtensions.OutputTypes;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+
+namespace SampleDriver
+{
+    class Program
+    {
```

Comment:
Can we add a quick docstring explaining what this does and what parts of the SDK it utilizes?